### ISSUE TITLE
Ability to load a game from battle (UX refinement)

### ISSUE THREAD

**Issue by dtcyganok** [ihhub/fheroes2#9337]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Sometimes the player gets stuck in a hopeless fight with no way out and is forced to wait for the battle to end.

### Describe the possible solution

Having the new button in battle mode that allows to load a game

### Additional info

Heroes 3 HotA offers such a feature for a long time. And to me it seems one of the most missing UX features. A great many of gamers would benefit from it.

**Comment by oleg-derevenetz** [comment]
The player does not have to wait for the battle to end. Just press the Q button, confirm your decision, and the battle will finish instantly.

**Comment by dtcyganok** [comment]
That's cool! I didn't know that.

**Comment by ihhub** [comment]
Loading a save file from battle seems illogical and the current UI simply doesn't have space for new buttons. As @oleg-derevenetz pointed a player can end a battle instantly.

**Comment by dtcyganok** [comment]
> Loading a save file from battle seems illogical and the current UI simply doesn't have space for new buttons. As @oleg-derevenetz pointed a player can end a battle instantly.

Yeah, the hotkey Q and the new IU solve 80% of the original problem.
But still, there is often a need to load a game after a bad decision. And having such a button would save a few clicks.

I don't insist. You decide for yourself whether it's worth your time and effort or not.

**Comment by ihhub** [comment]
Moving this issue into the discussion section as it is beyond the scope of the project.
